dev-docs/modules/jwplayerRtdProvider.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: page_v2
+title: JW Player Real Time Data Provider
+description: makes JW Player's video ad targeting information accessible 
+             to Bid Adapters.
+page_type: module
+module_type: rtd
+module_code : jwplayer
+enable_download : true
+sidebarType : 1
+---
+
+# JW Player RTD Provider
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+The purpose of this Real Time Data Provider is to allow publishers to target against their JW Player media without 
+being forced to integrate with the Player Bidding product. This prebid module makes JW Player's video ad targeting information accessible 
+to Bid Adapters.
+
+## Implementation for Publishers:
+
+1) Compile the JW Player RTD Provider into your Prebid build:
+
+`gulp build --modules=jwplayerRtdProvider`
+
+2) Publishers must register JW Player as a Real Time Data provider by using `setConfig` to load a Prebid Config containing a `realTimeData.dataProviders` array:
+
+```javascript
+const jwplayerRtdProvider = {
+  name: "jwplayer"
+};
+
+pbjs.setConfig({
+    ...,
+    realTimeData: {
+      auctionDelay: 1000,
+      dataProviders: [
+          jwplayerRtdProvider
+      ]
+    }
+});
+``` 
+
+3) In order to prefetch targeting information for certain media, include the media IDs in the `jwplayerRtdProvider` var before calling `setConfig`:
+
+```javascript
+const jwplayerRtdProvider = {
+  name: "jwplayer",
+  params: {
+    mediaIDs: ['abc', 'def', 'ghi', 'jkl']
+  }
+};
+```
+
+**Note:** setting an `auctionDelay` in the `realTimeData` object is required to ensure the auction waits for prefetching to complete. The `auctionDelay` is the max time in ms that the auction will wait for the requested targeting information.
+
+**Config Syntax details:**
+
+{: .table .table-bordered .table-striped }
+| Name  |Type | Description   | Notes  |
+| :------------ | :------------ | :------------ |:------------ |
+| name | String | Real time data module name | Always 'jwplayer' |
+| params | Object | | |
+| params.mediaIDs | Array of Strings | Media Ids for prefetching | Optional. |
+
+4) Include the content's media ID and/or the player's ID in the matching AdUnit before calling `addAdUnits`:
+
+```javascript
+   const adUnit = {
+     code: '/19968336/prebid_native_example_1',
+     ...
+     jwTargeting: {
+       waitForIt: true,
```

Comment:
@karimMourra - I don't think this is right.

1) I don't like the idea of arbitrarily adding `jwTargeting` to the AdUnit object. I think you should be posing this as [First Party Data](https://docs.prebid.org/features/firstPartyData.html). e.g. fpd.context.jwTargeting would be ok.
2) `waitForIt` is part of the `RealTimeData` setConfig, not part of the AdUnit.

Reply:
@bretg  addressed